stories/queryWizard/examples.stories.js

```diff
@@ -0,0 +1,81 @@
+import _ from 'lodash/fp'
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import {
+  QueryWizard as GVQueryWizard,
+  FilterButtonList as GVFilterButtonList,
+  StepsAccordion as GVStepsAccordion,
+} from '../../src/themes/greyVest'
+import DefaultQueryWizard from '../../src/queryWizard/QueryWizard'
+import DefaultStepsAccordion from '../../src/layout/StepsAccordion'
+import DefaultFilterButtonList from '../../src/FilterButtonList'
+import { mergeOverAll } from '../../src/utils/futil'
+import { componentForType } from '../../src/utils/schema'
+import GVDecorator from '../greyVest/decorator'
+import { ExampleTypes } from '../DemoControls'
+import { tree, fields, types, nodeOverrides } from './config'
+let { TypeMap } = ExampleTypes
+
+let mapNodeToDescription = types => (node, fields) => ({
+  description: _.join(' ', [
+    _.get([node.field, 'description'], fields) || node.description,
+    _.get([node.type, 'description'], types),
+  ]),
+})
+
+let story = QueryWizard => () => (
+  <QueryWizard
+    tree={tree}
+    path={['root']}
+    fields={fields}
+    mapNodeToProps={componentForType(TypeMap)}
+    mapNodeToLabel={(node, fields) => _.get([node.field, 'label'], fields)}
+    mapNodeToDescription={mapNodeToDescription(types)}
```

Comment:
These shouldn’t still be here

Reply:
so let's remove them :)

Reply:
sorry - I could swear I removed these locally at one point, and they somehow snuck back in

it is a mystery 👻